File: src/containers/desafio11POO.js
//crea una funcion que reciba dos cadenas como parametro (str1, str2)
// e imprima otras dos cadenas como salida (out1, out2). out1 contendra todos los caracteres
//presentes en la str1 pero no esten presentes en str2. out2 contendra todos los caracteres presentes
// en la str2 pero no esten presentes en str1 
class Palabra{
    constructor(str1,str2){
        this.str1=str1;
        this.str2=str2;
    }
    cadena1(){
        lst1=[]
        for(let letra in this.str1){
            if (letra in this.str2){
                return false;
            }
            else {
                lst1.push(this.letra)
            }
        }
        console.log(lst1.join(" "))
    }
}
cadena= new Palabra("felicidades", "programacion")
console.log(cadena.cadena1())

File: src/containers/desafio11POO.py
class cadenas:
    def __init__(self,str1,str2):
        self.str1= str1
        self.str2= str2
        self.lst1=[]
        self.lst2=[]
    def cadena1(self,str1, str2):
        for letra in self.str1:
            if letra not in self.str2:
                self.lst1.append(letra)
        print("".join(self.lst1)) 
    
    def cadena2(self,str1,str2):
        for letra in self.str2:
            if letra not in self.str1:
                self.lst2.append(letra)            
        print("".join(self.lst2))   

caden= cadenas("felicidades","programacion")
print(caden.cadena1("felicidades","programacion"))
print(caden.cadena2("felicidades","programacion"))
